katas/content/superposition/all_basis_vectors/Solution.qs

```diff
@@ -0,0 +1,8 @@
+namespace Kata {
+    
```

Comment:
I like to not have empty line between namespace and first operation (unless there are open statements which deserve a line after to separate them from operation - but no empty line after namespace) for the sake of keeping code compact (not a lot of space if you view the modern katas on a laptop screen)
```suggestion
```